_support-articles/known-incompatible-software.html

```diff
@@ -161,12 +168,26 @@
 
       The latest version of D3DGear have fixed this problem. To fix this problem, uninstall D3DGear and [install the latest version](http://www.d3dgear.com/D3DGearSetup.exe).
 
+  - name: Discord
+    versions: All Versions
+    notes: |
+      Causes freezes or crashes.
+
+      Disable the Overlay feature in Discord.
+  
+  - name: Evolve and/or Player.me
+    versions: All Versions
+    notes: |
+      Causes freezes or crashes.
+
+      Disable the the Ingame-Overlay feature.
+
   - name: GeForce Experience
     versions: All Versions
     notes: |
-      Causes keys to become stuck in Minecraft.
+      Causes keys to become stuck in Minecraft. As well as causing some crashes.
```

Comment:
The as well as feels out of place, should just be a single sentence or phrased differently

Reply:
Both changed.